ci/python_bandit_check.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -uex
+
+rflg=
+gitfiles=()
+if [ -e '.git' ]; then
+  read -ra gitfiles < <(git ls-tree --name-only HEAD)
+  if [ "${#gitfiles[@]}" -ne 0 [; then
```

Comment:
(lint) The mentioned parser error was in this if expression. [SC1009]
(lint) Couldn't parse this test expression. [SC1073]
(lint) Expected "]". Fix any mentioned problems and try again. [SC1072]